docs/03.reference/02.tags/case/_examples.md

```diff
@@ -7,16 +8,24 @@
     <cfcase value=2$3$4 delimiters="$">
         this is from case 2
     </cfcase>
+    <cfcase value=5,6,7>
+        this is from case 3
+    </cfcase>
     <cfdefaultcase>
         this is from default part
     </cfdefaultcase>
 </cfswitch>
 
+<!--- Script syntax --->
 <cfscript>
-    switch(1){
+    expr = 1;
+    switch(expr){
         case 1:
             result = 1;
             break;
+        case 2: case 3:
```

Comment:
can you put each case on a new line?

Reply:
I've changed the file, but I cannot "Commit directly to the patch-1 branch". The commit button stays inactive.
Sorry, I really have no experience with PRs, commiting, etc..